governance/env-drift.py

```
@@ -0,0 +1,188 @@
+    return {"drifts": drifts, "skipped": skipped, "checked": checked}
+# @w4r1-pure-end
+
+
+def main():
+    import tempfile
+    actions = {"scope": [], "checked": 0, "drifts": 0}
+    try:
+        policy = load_yaml(POLICY)
+        scope = policy.get("scope") or []
+        if not scope:
+            print("FATAL policy/env-drift.yaml 缺 scope（检测面旋钮——AC-8b）", file=sys.stderr)
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)